src/standalone/Bucket/Bucket.jl

```diff
@@ -452,6 +440,59 @@ function make_update_aux(model::BucketModel{FT}) where {FT}
             p,
             t,
         )
+
+        #Currently, the entire surface is assumed to be
+        # snow covered entirely or not at all.
+        p.bucket.snow_cover_fraction .= heaviside.(Y.bucket.σS)
+
+        # In this case, there is just one set of surface fluxes to compute.
+        # Since q_sfc itself has already been modified to account for
+        # snow covered regions, and since the surface temperature is
+        # assumed to be the same for snow and underlying land,
+        # the returned fluxes are computed correctly for the cell
+        # regardless of snow-coverage.
+
+        # The below must be adjusted to compute F_sfc over snow and over soil
+        # if we want the snow cover fraction to be intermediate between 0 and 1.
+        @. p.bucket.F_sfc = (
```

Comment:
Should we add a test for this? E.g. here
https://github.com/CliMA/ClimaLand.jl/blob/97f06a17e349b8666b6b73d89657b4bdaae03c4a/test/standalone/Bucket/soil_bucket_tests.jl#L216

Reply:
Added